README.md

```diff
@@ -566,6 +566,7 @@ The MARIN3R mutating admission webhook will inject Envoy containers in any Pod a
 | marin3r.3scale.net/shutdown-manager.port     | Envoy's shutdown manager server port                                                                                                                                                                           | 8090                                                     |
 | marin3r.3scale.net/shutdown-manager.image    | Envoy's shutdown manager image                                                                                                                                                                                 | If unset, the operator will select the appropriate image |
 | marin3r.3scale.net/init-manager.image        | Envoy's init manager image                                                                                                                                                                                     | If unset, the operator will select the appropriate image |
+| marin3r.3scale.net/init-manager.image        | Comma separated list of container names whose stop should be coordinated with the shutdown-manager. You usually would want to add containers that act as upstream clusters for the Envoy sidecar               | N/A                                                      |
```

Comment:
duplicated var, should be ` marin3r.3scale.net/shutdown-manager.extra-lifecycle-hooks`?

Reply:
definitely :sweat_smile:

Reply:
Fixed c472633